src/main/java/pl/com/tenderflex/service/UserService.java

```diff
@@ -0,0 +1,29 @@
+package pl.com.tenderflex.service;
+
+import org.springframework.dao.DataAccessException;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.stereotype.Service;
+import pl.com.tenderflex.dao.UserDao;
+import pl.com.tenderflex.exception.ServiceException;
+import pl.com.tenderflex.exception.UserNotFoundException;
+import pl.com.tenderflex.model.User;
+
+@Service
+public class UserService {
+
+    private final UserDao userDao;
+
+    public UserService(UserDao userDao) {
+        this.userDao = userDao;
+    }
+
+    public User getByEmail(String email) {
+        try {
+            return userDao.getByEmail(email);
+        } catch (EmptyResultDataAccessException e) {
```

Comment:
Such Exceptions could be handled in Controller Adviser. No need to wrap every Exception. It would make code clear